CHANGELOG.md

```diff
@@ -14,6 +14,21 @@ All notable changes for each version of this project will be documented in this
     - `IgxHierarchicalGridBaseComponent` -> `IgxHierarchicalGridBaseDirective`
     - `IgxMonthPickerBase` -> `IgxMonthPickerBaseDirective`
 
+- **Breaking Changes** Due to a breaking change in Angular 9 Hammer providers are no longer included by default. You can find more information at: https://github.com/angular/angular/blob/master/CHANGELOG.md#breaking-changes-9 . Because of this change the following components require `HammerModule` to be imported in the root module of the application in order for user interactions to work as expected:
```

Comment:
This is Ivy-specific I think (https://next.angular.io/guide/ivy-compatibility#less-common-changes). Still the default, but might be worth noting.